content/en/user-guide/lambda-tools/debugging/index.md

```diff
@@ -184,7 +184,7 @@ $ awslocal lambda create-function --function-name my-cool-local-function \
 
 We can quickly verify that it works by invoking it with a simple payload:
 
-{{< tabpane text=true persistLang=false >}}
+{{< tabpane text=true persist=false >}}
```

Comment:
lang not needed here?

Reply:
The API changed here, yes.

Reply:
Ah, I think I misunderstood. No, lang is not necessary here. It is necessary to default the persistence key to the tab headers, but since persistence is disabled here, it does not matter.